### PR TITLE
Commit without haste

### PR DESCRIPTION
**Pull request by KPrasch**
* Immediate help for issues that cause nodes to crash
* Don't leave the cursor hanging, polling the provider for a receipt and confirmations.  
* Allows the work tracker to track transaction hashes instead of receipts, by broadcasting commitment transactions when they are indicated by the staking contract without waiting for a receipt.
* Removes default work tracker qualification callable until #2277 is resolved. (`__requirement`)
* Advances #2070 #1852 #2141 

Additional follow-up in #2141

### CODE REVIEW

File: nucypher/blockchain/eth/interfaces.py [hunk 1]
```diff
@@ -631,7 +645,8 @@ def send_transaction(self,
 
         receipt = self.sign_and_broadcast_transaction(transaction_dict=transaction,
                                                       transaction_name=transaction_name,
-                                                      confirmations=confirmations)
+                                                      confirmations=confirmations,
+                                                      fire_and_forget=fire_and_forget)
         return receipt
```
Comment (derekpierre) [inline nucypher/blockchain/eth/interfaces.py:650]: We should use `txhash_or_receipt` here instead of receipt

File: nucypher/characters/lawful.py [hunk 2]
```diff
@@ -1217,7 +1217,7 @@ def run(self,
                 emitter.message(f"✓ Availability Checks", color='green')
 
         if worker and not self.federated_only:
-            self.work_tracker.start(act_now=True, requirement_func=self._availability_tracker.status)
+            self.work_tracker.start(act_now=True)  # requirement_func=self._availability_tracker.status)  # TODO: #2277
```
Comment (derekpierre) [inline nucypher/characters/lawful.py:1220]: 👍

**Review by derekpierre — APPROVED** [review]
🎸 - excited to see this in the wild. Just one comment about a variable name.

File: nucypher/blockchain/eth/interfaces.py (hunk 1 above)
Comment (KPrasch) [reply nucypher/blockchain/eth/interfaces.py:650]: :heavy_check_mark:

File: nucypher/blockchain/eth/interfaces.py [hunk 3]
```diff
@@ -529,9 +530,18 @@ def build_contract_transaction(self,
...
-                                       ) -> dict:
+                                       confirmations: int = 0,
+                                       fire_and_forget: bool = False
+                                       ) -> Union[TxReceipt, HexBytes]:
+        """
+        Takes a transaction dictionary, signs it with the configured signer, then broadcasts the signed
+        transaction the ethereum provider's eth_sendRawTransaction RPC endpoint.
+        Optionally blocks for receipt and confirmation with 'confirmations', and 'fire_and_forget' flags.
+
+        If 'fire and forget' is True this method returns the transaction hash only -
```
Comment (cygnusv) [inline nucypher/blockchain/eth/interfaces.py:541]:
```suggestion
        transaction using the ethereum provider's eth_sendRawTransaction RPC endpoint.
        Optionally blocks for receipt and confirmation with 'confirmations', and 'fire_and_forget' flags.

        If 'fire and forget' is True this method returns the transaction hash only, without waiting for a receipt -
```

**Review by cygnusv — APPROVED** [review]
Great! One concern less :D

**Pull request merged**